src/components/breadcrumbs/crumb/crumb.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Crumb } from "./crumb.component";
+
+const LINK_TEXT = "Link text";
+
+describe("Crumb", () => {
+  it("passes href to the anchor element isCurrent is false", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} />);
+
+    const link = screen.getByText(LINK_TEXT);
+    expect(link).toBeInTheDocument();
+    expect(link.closest("a")).toHaveAttribute("href", "foo");
+  });
+
+  it("does not pass href to the anchor element when isCurrent is true", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} isCurrent />);
+
+    const anchor = screen.getByText(LINK_TEXT).closest("a");
+    expect(anchor).not.toHaveAttribute("href", "foo");
+
+    expect(anchor).toHaveStyle({
+      color: "var(--colorsYin055)",
+      textDecoration: "none",
+      cursor: "text",
+    });
+  });
+
+  it("calls the onClick callback if one is passed and isCurrent is false", async () => {
+    const onClickFn = jest.fn();
+
+    render(<Crumb href="foo" text={LINK_TEXT} onClick={onClickFn} />);
```

Comment:
Currently we get the following warning when these click callback tests are run:

```shell
Error: Not implemented: navigation (except hash changes)
          at module.exports (/Users/james.parslow/Development/carbon/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
          at navigateFetch (/Users/james.parslow/Development/carbon/node_modules/jsdom/lib/jsdom/living/window/navigation.js:76:3)
          at exports.navigate (/Users/james.parslow/Development/carbon/node_modules/jsdom/lib/jsdom/living/window/navigation.js:54:3)
          at Timeout._onTimeout (/Users/james.parslow/Development/carbon/node_modules/jsdom/lib/jsdom/living/nodes/HTMLHyperlinkElementUtils-impl.js:81:7)
          at listOnTimeout (node:internal/timers:564:17)
          at processTimers (node:internal/timers:507:7) undefined
```

This is because [jsdom does not support window navigation](https://github.com/jsdom/jsdom/issues/2112#issuecomment-359297866) and it is seeing `foo` as a url to navigate to. To get around this, I would recommend making the `Crumbs` in these tests a dummy link:

```suggestion
    render(<Crumb href="#" text={LINK_TEXT} handleClick={handleClickFn} />);
```

That way you can still test the click callback behaviour, but jsdom won't try to redirect to a new page when the link is clicked.